tools/flang1/flang1exe/semfunc.c

```diff
@@ -4178,6 +4228,9 @@ ref_intrin(SST *stktop, ITEM *list)
   int tmp, tmp_ast;
   FtnRtlEnum rtlRtn;
   int intrin; /* one of the I_* constants */
+  int dtype_new = 0;
```

Comment:
`dtype_new` is not used after it is set by `check_max_min_argument`. Can we delete it?

Reply:
`dtype_new` can be deleted, but `dtype_last` needs to record the last value（The type of the previous argument） and compare it with the new value, So I haven't thought of a good way to delete `dtype_last` yet.

Reply:
OK